config/tekton-dashboard.yaml

```diff
@@ -0,0 +1,59 @@
+# ------------------- Dashboard Deployment ------------------- #
```

Comment:
I'm being a bit fussy now but the Deployment section of this file shouldn't be showing any changes. Is this just git getting confused or have you moved and recreated the file?

Reply:
Maybe I had an old copy of this - it's looking much cleaner now.